docs/concepts/Deploy-Command.md

```diff
@@ -0,0 +1,80 @@
+# Deploy Command
+
+`askx deploy` deploys all the Alexa skill project resources by following the settings from the project config file (i.e. ask-resources.json). It will deploy each [resource component](./Alexa-Skill-Project-Definition.md) from the local file(s) to the target endpoint sequentially. The deploy flow for each component has been optimized to be executed only when new changes exist in the local (see [usage of lastDeployHash]()). Users of deploy command can focus more on the content within each component, and only control the deployment when it's necessary. 
```

Comment:
link needs to be updated for `resource component`, `usage of lastDeployHash` and `here` (about ask-resources config file).

Reply:
The resource component one works only when you combine them together, as it's a relative path. I will add the usage of lastDeployHash and link it to the sentence.